packages/kbn-storybook/src/lib/default_config.ts

```diff
@@ -43,3 +46,14 @@ export const defaultConfig: StorybookConfig = {
     return emotion11CompatibleConfig;
   },
 };
+
+export const defaultConfigWebFinal = {
+  ...defaultConfig,
+  webpackFinal: (config: Configuration) => {
+    return WebpackConfig({ config });
+  },
+};
+
+export const mergeWebpackFinal = (extraConfig: Configuration) => {
+  return { webpackFinal: (config: Configuration) => webpackMerge(config, extraConfig) };
+};
```

Comment:
i moved these to storybook package because webpackFinal usage in storybook main.ts somehow is causing issues with newly added dependency of ts-node most likely an issue with storybook typescript setup

storybookjs/storybook#9610

Reply:
Mind adding a comment here with a link explaining that for who comes across it next?

Reply:
Sure, i will add those